components/services/markdown/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+const STATUS_OK = 200
+
+class ServiceMarkdown extends Component {
+  state = {html: ''}
+  async componentDidMount() {
+    const marked = await import('marked')
+    const req = new window.XMLHttpRequest()
+    req.open('GET', this.props.src, false)
```

Comment:
If we do an HTTPRequest to get the statics we should handle the possibility of a exception throw due to CORS incompatibility, for example (this happened to me testing it on surge).

On the other hand I feel that the MD load is a bit slow in some cases and we are not adding any feedback to the user. Is this a right behavior